src/test/javascript/spec/component/orion/orion-code-editor-instructor-and-editor-container.component.spec.ts

```diff
@@ -57,41 +61,46 @@ describe('CodeEditorInstructorAndEditorOrionContainerComponent', () => {
                 orionConnectorService = TestBed.inject(OrionConnectorService);
             });
     });
+
     afterEach(() => {
-        sinon.restore();
+        jest.restoreAllMocks();
     });
 
     it('applyDomainChange should inform connector', () => {
-        const selectRepositorySpy = spy(orionConnectorService, 'selectRepository');
+        const selectRepositorySpy = jest.spyOn(orionConnectorService, 'selectRepository');
 
         // @ts-ignore
         comp.applyDomainChange({}, {});
 
-        expect(selectRepositorySpy).to.have.been.calledOnceWithExactly(REPOSITORY.TEST);
+        expect(selectRepositorySpy).toHaveBeenCalledTimes(1);
+        expect(selectRepositorySpy).toHaveBeenCalledWith(REPOSITORY.TEST);
     });
     it('ngOnInit should subscribe to orionState', () => {
-        const orionStateStub = stub(orionConnectorService, 'state');
+        const orionStateStub = jest.spyOn(orionConnectorService, 'state');
         const orionState = { opened: 40, building: false, cloning: false } as any;
-        orionStateStub.returns(new BehaviorSubject(orionState));
+        orionStateStub.mockReturnValue(new BehaviorSubject(orionState));
```

Comment:
I suggest ln 1: Define orionState, ln 2: define the stub => the stub can be one line instead of two

Reply:
I compressed all occurences of this :+1: